openpype/hosts/blender/hooks/pre_add_run_python_script_arg.py

```diff
@@ -0,0 +1,53 @@
+from pathlib import Path
+
+from openpype.lib import PreLaunchHook
+
+
+class AddPythonScriptToLaunchArgs(PreLaunchHook):
+    """Add python script to be executed before Blender launch."""
+
+    # Append after file argument
+    order = 15
+    app_groups = [
+        "blender",
+    ]
+
+    def execute(self):
+        if not self.launch_context.data.get("python_scripts"):
+            return
+
+        # Add path to workfile to arguments
+        for python_script_path in self.launch_context.data["python_scripts"]:
+            self.log.info(
+                f"Adding python script {python_script_path} to launch"
+            )
+            # Test script path exists
+            python_script_path = Path(python_script_path)
+            if not python_script_path.exists():
+                raise ValueError(
```

Comment:
Shouldn't we still allow to launch the dcc and maybe log it as a warning?

Reply:
I can change that next week, sure.